src/Razor/src/Microsoft.VisualStudio.RazorExtension.Dependencies/Microsoft.VisualStudio.RazorExtension.Dependencies.csproj

```diff
@@ -68,7 +69,7 @@
     <PackageReference Include="Microsoft.VisualStudio.LanguageServices.Implementation.Symbols" Version="$(Tooling_MicrosoftVisualStudioLanguageServicesPackageVersion)" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(IncludeRoslynDeps)' == 'true'">
```

Comment:
📝 It wasn't clear why we need this flag. I've removed it for now, but maybe we need it for something?

Reply:
FWIW, it was added in https://github.com/dotnet/razor/pull/8086